src/applications/dhp-connected-devices/components/FAQSections.jsx

```diff
@@ -259,44 +259,22 @@ export const ThirdFAQSection = () => {
           below:
           <ul>
             <li>
-              Login.gov - Access the Login.gov help center at{' '}
-              <va-telephone
-                contact="8448756446"
-                aria-label="8 4 4. 8 7 5. 6 4 4 6."
-                international
-              />
+              <strong>Login.gov</strong> - Access the Login.gov help center at{' '}
+              <va-telephone contact="8448756446" international />
             </li>
             <li>
-              ID.me - Go to the ID.me help center at{' '}
-              <va-telephone
-                contact="8559274363"
-                aria-label="8 5 5. 9 2 7. 4 3 6 3."
-                international
-              />
+              <strong>ID.me</strong> - Go to the ID.me help center at{' '}
+              <va-telephone contact="8559274363" international />
             </li>
             <li>
-              DS Logon - Call the DMDC Support Office at{' '}
-              <va-telephone
-                contact="8005389552"
-                aria-label="8 0 0. 5 3 8. 9 5 5 2."
-                international
-              />
+              <strong>DS Logon</strong> - Call the DMDC Support Office at{' '}
+              <va-telephone contact="8005389552" international />
             </li>
             <li>
-              My HealtheVet - Contact the My HealtheVet Help Desk at{' '}
-              <va-telephone
-                contact="8773270022"
-                aria-label="8 7 7. 3 2 7. 0 0 2 2."
-                international
-              />{' '}
-              or{' '}
-              <va-telephone
-                contact="8008778339"
-                aria-label="TTY. 8 0 0. 8 7 7. 8 3 3 9."
-                tty
-                international
-              />{' '}
-              (TTY), Monday to 7:00 a.m. — 7:00 p.m. (Central Time)
+              <strong>My HealtheVet</strong> - Contact the My HealtheVet Help
+              Desk at <va-telephone contact="8773270022" international /> or{' '}
+              <va-telephone contact="8008778339" tty international /> (TTY),
```

Comment:
Does this fix the screen reader duplication issue? And does it still read TTY out loud?

Reply:
Yes we added an aria-label but the `va-telephone` component already creates one.

Reply:
So that's why it duplicated. Yes it does.